assets/src/dashboard/animations/bounce/index.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+function Bounce({ style, duration, delay, children }) {
```

Comment:
Here's one suggestion (and there are some related comments in other places). What if this was just a single `Animated` component that would accept the props based on the [Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Element/animate). E.g. these props would include: `delay`, `duration`, `easing`, `fill`, other options, and animation name (which will be resolvable to `keyframes` CSS and/or object declaration). We could even go one step further and allow an array with this payload, since animations can be mixed on the same wrapper as long as they don't conflict on the same property (which is easy enough to check). We can still accept the initial `style`, but in most of cases, we can simply look into keyframes definition for the offset=0 and pick the initial styles from there automatically.

So the bottom line, we could follow [Element.animate](https://developer.mozilla.org/en-US/docs/Web/API/Element/animate) API very closely. That gives us a well-established backing of the existing API and, if we are careful enough, we can take advantage of both: CSS and API animations for different environments.

Reply:
Yes, this makes sense to me!  I'll go ahead and make this refactor and ping you for a re-review when it's up!